box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsFile.java

```diff
@@ -1159,4 +1162,52 @@ protected void onSendCompleted(BoxResponse<BoxVoid> response) throws BoxExceptio
             super.handleUpdateCache(response);
         }
     }
+
+    public static class DownloadRepresentation extends BoxRequestDownload<BoxDownload, DownloadRepresentation> {
```

Comment:
Curious to know how does pagination work here? Upon executing it in its default form I guess i will get back the response for page 1 if pagination is supported. To get page 2, is it as simple as doing `setRequestPage(2)` and executing the request again?

Reply:
yes, if the request is paged, then you have to call again for every page. If the request is not paged, then you either get the whole document (pdf case), or the first page only (images)